executor/install/bin/post-install.py

```
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+
+# NOTE: This script is a wrapper for the actual post-install script.
+
+import logging
+
+logging.basicConfig(level=logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+from pathlib import Path
```

Ruff (0.12.2)

Module level import not at top of file

(E402)